scenarios/ecs_privesc_evade_protection/assets/flag.txt

```diff
@@ -0,0 +1 @@
+cg-secret-bob12-gC9!+Xy#QJ37fa@H3D7Kd@2*a&#+Tp%
```

Comment:
Assets used by the scenario should be moved to under the terraform directory. Like [done here](https://github.com/RhinoSecurityLabs/cloudgoat/tree/master/scenarios/vulnerable_lambda)